ethereum/eip712/message.go

```diff
@@ -100,6 +100,10 @@ func getPayloadMessages(payload gjson.Result) ([]gjson.Result, error) {
 		return nil, errorsmod.Wrap(errortypes.ErrInvalidRequest, "no messages found in payload, unable to parse")
 	}
 
+	if rawMsgs.Type == gjson.Null {
```

Comment:
Was this giving issues?

Reply:
yes, many interfaces changed on the sdk v0.50. Previously when passing empty messages (`[]sdk.Msg{}`) to the txBuilder, the `rawMsgs` would return an empty array. Thus this function is not returning an error.
With the sdk v0.50, when receiving emtpy messages, the txBuilder sets Msgs = nil. Thus, `rawMsgs` is null, and the function fails because null is not an Array. The line added handles this case